Cleaning up the Readme (#33)
SuperDARN/pydarn: docs/user/superdarn_data.md

# Getting SuperDARN Data 

pyDARN does not provide an interface to downloading data. However, there are other means of getting access to the data. 

> Data Distribution Working Group manages the checking and distribution of data if you have any inquiries, please contact the head chair Kevin Krieger at superdarn@usask.ca

Reviewer: ```suggestion
Data Distribution Working Group manages the checking and distribution of data if you have any inquiries, please contact the chair Kevin Krieger at superdarn@usask.ca
```

## Data Mirrors

> To get access to the data, there are two data servers setup: 

Reviewer: ```suggestion
To get access to the data, there are two possible data servers: one utilizes Globus, and one uses rsync, sftp and scp: 
```

  - [SuperDARN Canada](https://superdarn.ca/): uses [Globus](https://github.com/SuperDARNCanada/globus) to allow access to the SuperDARN data. 

>   - [BAS](https://www.bas.ac.uk/project/superdarn/#about): information on data access can be found [here](https://www.bas.ac.uk/project/superdarn/#about)

Reviewer: ```suggestion
  - [BAS](https://www.bas.ac.uk/project/superdarn/#about): information on data access can be found [here](https://www.bas.ac.uk/project/superdarn/#data)
```

> ## Citing Data

Reviewer: ```suggestion
```

> Please cite SuperDARN appropriately and/or pyDARN if used in any scientific publications. More information can be found here.

Reviewer: ```suggestion
```
